health/rpc/api.go

```diff
@@ -0,0 +1,40 @@
+package rpc
```

Comment:
Trying a slightly different directory structure in the new modules where all rpc things (.proto file, generated code, api impl) go in a `rpc` directory. I think I like it.

Reply:
Yeah, the `pb` folder not being inside sounds purist, but in reality is only used for grpc.
I like that too, less folders is good.